packages/clay-table/src/ClayTable.soy

```diff
@@ -246,7 +246,15 @@
 	{@param? showActionsMenu: bool}
 	{@param? spritemap: string}
 
-	<tr data-onclick="{$_handleRowContentClick}">
+	{let $attributes kind="attributes"}
+		{if $selectable and $item.selected}
```

Comment:
not only on selectable, this should work always (even when there's no checkbox present, that's the use case for commerce). 

If you find it weird maybe we can implement something like `showCheckbox`  and use `selectable` as you pointed.

@FabioDiegoMastrorilli sounds good?

Reply:
@carloslancha the second idea is the best: we do need to pass `selectable` as a parameter in order to emit an event but, at the moment, it also affect the html markup (showing a checkbox).
The fact is that we want the row to be selected but we don't need the checkbox

Thank you guys :)

Reply:
I'm going to add the `showCheckbox` API but keep the `selectable` for the context of whether or not the row is selectable. Thank you guys!